publish/working-with-stacks/working-with-stacks.md

```diff
@@ -217,66 +217,94 @@ Now that your stack is available in your local repository, the next step is to r
+  - Developers who are using an IDE with the Eclipse Codewind extension can configure their template sources. For more information, see [Getting Started with Codewind and VSCode](../codewind-getting-started-vscode/) or [Getting Started with Codewind and Eclipse](../codewind-getting-started-eclipse/).
 
-## Configuring the Kubernetes operator custom resource definition
+Developers can now create applications based on your customized stack in their local development environment. If you plan to make more than one  application stack available to developers, you might want to consider creating a stack hub. For more information, see [Creating a stack hub](../creating-a-stack-hub/).
 
-In order to deploy containerized applications that have been developed using your application stack, you must configure Kubernetes with information about the application stack and the deployment container that must be used. This configuration forms part of the operator custom resource definition (CRD).
+## Configuring your Kubernetes environment
 
-The following section shows the configuration for stack repositories:
+In order to deploy containerized applications that have been developed using your application stack, you must configure Kubernetes with information about the application stack and the deployment container that must be used. This configuration forms part of the Kabanero operator custom resource definition (CRD). The process is the same whether your URL defines a single application stack or a stack hub that contains multiple application stacks.
 
-```
-stacks:
-  repositories:
-  - name:
-    https:
-      url:
-      skipCertVerification: [true|false]
-  - name:
-    https:
-      url:
-      skipCertVerification: [true|false]
-```
+Follow these steps:
 
-Where:
+1. Obtain a copy of your current Kabanero CR instance configuration.
 
-- `repositories` lists the repositories to search for stacks
-- `name` is the name of your repository
-- `url` contains the URL string for the `index.yaml` file
-- `skipCertVerification` determines whether to verify the certificate before activating (default is `false`)
+    To obtain a list of all Kabanero CR instances in the kabanero namespace, run the following command:
 
-To change the desired state for a repository after the instance is deployed, you must edit each resource manually.
+    ```
+    oc get kabaneros -n kabanero
+    ```
 
-The following configuration can be used for the repository you created earlier:
+    To obtain the configuration for a specific CR instance, run the following command, substituting `<name>` for the instance you are targeting:
 
-```
-stacks:
-  repositories:
-  - name: my-org-repository
-    https:
-      url: https://github.com/myorg/my-org-repository/releases/latest/download/my-org-repository-index.yaml
-```
+    ```
+    oc get kabanero <name> -o yaml > kabanero.yaml
```

Comment:
oc get kabanero <name> -n kabanero -o yaml > kabanero.yaml

Reply:
Fixed. Thank you! Could you please approve your review?